src/configurator/validators/assets.js

```diff
@@ -55,10 +59,11 @@ export default function(
         }
       }
 
+      const configAssets: PhenomicAssetsConfig = config.assets
```

Comment:
We can't assign `configAssets` at the beginning of that function.Because we adjust `configAssets` above this line. Flow doesn't understand the situation here.